Automattic/wp-calypso
# Emails: Group Email Comparison and In-Depth Comparison components logically

This pull request moves several components used by the new `Email Comparison` page as well as the `In-Depth Comparison` page under the `client/my-sites/email/email-providers-comparison` folder. The goal is to be able to share some of those components between those two pages, and to group components logically:

![image](https://user-images.githubusercontent.com/594356/148819227-a7a70177-7790-42b2-b950-eb129d684536.png)

This pull request also moves the `In-Depth Comparison` page under that folder but it doesn't touch the new `Email Comparison` page given that we have other refactorings going on (we can move it once they have been merged). Apart from relocating code and extracting components into their own folder, I barely touched any logic. I renamed some CSS classes but that's all.

#### Testing instructions

1. Run `git checkout move/email-comparison-components` and start your server, or access a [live branch](https://github.com/Automattic/wp-calypso/pull/59916#issuecomment-1009171900)
2. Log into a WordPress.com account with a domain but no email
3. Open the [`Emails` page](http://calypso.localhost:3000/email)
4. Click the `Add Email` button of that domain to access the `Email Comparison` page
5. Assert that the page works as before
6. Click the `See how they compare` link to access the `In-Depth Comparison` page
7. Assert that the page works as before
